src/client/components/appheader/appheader.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
+import logo from './logoheader.png';
+
+import './appheader.css';
+
+const volumeIcon = <FontAwesomeIcon icon={faUser} />;
+
+const AppHeader = () => {
+  return (
```

Comment:
Here you could write component without return statement and curly bracers, like this
const AppHeader = () => ( 
);